components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
```

Comment:
We prefer to use no arrow functions.

```suggestion
export default function Button({ setProduct, produkt, productToBuy }) {
```